File: google/test/src/utils/TestPrettifier.java
package test.src.utils;

import static org.junit.jupiter.api.Assertions.assertEquals;

import java.util.HashSet;

import org.junit.jupiter.api.Test;

import main.src.utils.Prettifier;


public class TestPrettifier {

    @Test
    public void testPrettifyHashSet() {
        HashSet<Integer> input = new HashSet<>();
        input.add(1);
        input.add(2);
        input.add(3);
        
        String actual = Prettifier.prettify(input);
        String expected = "\t1) 1\n\t2) 2\n\t3) 3\n";

        assertEquals(expected, actual);
    }
}

File: GoogleSharp/GoogleSharp.Test/Tests/src/utils/TestPrettifier.cs
using System;
using Xunit;
using Utils;
using System.Collections.Generic;

namespace GoogleSharp.Test
{
    public class TestPrettifier
    {
        [Fact]
        public void TestPrettifyHashSet()
        {
            var inputSet = new HashSet<string>();
            inputSet.Add("hello");
            inputSet.Add("bye");
            string actual = Prettifier<string>.Prettify(inputSet);
            string expected = "\t1) hello\n\t2) bye\n";
            Assert.Equal(expected, actual);
        }
    }
}
